app/src/main/java/com/venuenext/venuenextsdkdemo/MainActivity.kt

```diff
@@ -162,18 +190,37 @@ class MainActivity : AppCompatActivity(), BottomNavigationView.OnNavigationItemS
     }
 
     private fun completeInitialize() {
+
         registerFCM()
+        VNOrderUI.initialize()
+        VNWalletUI.initialize()
+        VNPayment.configurePaymentProcessing(BraintreePaymentProcessableFragment())
```

Comment:
(Optional) Maybe we should have this be a default since most integrattors use it. Anyone who wants to provide their own can just call this manually

Reply:
The Braintree fragment? I didn’t even realize we provided that. Thought it was just a file in both apps lol. K, yeah I agree with that.

Reply:
But that would be an SDK change so I'll make a ticket for it.

Reply:
Agreed. I probably should have done this as part of the new initialization work.